mkdocs/utils.py

```diff
@@ -35,6 +36,39 @@
     text_type = unicode         # noqa
 
 
+def yaml_load(source, loader=yaml.Loader):
+    """
+    Custom yaml loader.
+
+    Load all strings as unicode.
+    http://stackoverflow.com/a/2967461/3609487
+    """
+
+    class Loader(loader):
+
+        """Custom Loader."""
+
+        pass
```

Comment:
Actually I believe it would be an error to remove `pass` as nothing is defined within the class. Instead the class method `add_constructor` is called later (L58) which modifies the class. We can't just call `add_constructor` on the parent class (`yaml.Loader`) as that would modify the parent class (defined in the yaml lib) rather than a copy/ subclass. I agree, it is a little odd when reading the code, but the yaml lib's API requires it be that way. I suspect the comment was an attempt to clarify this (which it apparently did not do very well).